app/lib/sfd-client.js

```diff
@@ -0,0 +1,27 @@
+const sendMessage = require('../messaging/send-message')
+const { sfdRequestMsgType, sfdMessageQueue } = require('../config')
+const validateSFDClaim = require('../messaging/schema/submit-sfd-schema')
+const states = require('../messaging/application/states')
+
+const sendSFDEmail = async (templateId, email, personalisation) => {
+  const { personalisation: { reference, applicationReference, crn, sbi } } = personalisation
```

Comment:
I think like on the document generator, this deconstruct here ends up with us having too nested a structure like:
```
personalisation : {
  personalisation: {
    ...
  }
}
```

Reply:
We've tested review and follow-ups an neither had this nested structure in message to sfd proxy. We also tested email via gov notify and email looked correct.